src/actions/errors.js

```
@@ -8,15 +8,17 @@ export const TOGGLE_DETAILS = '@@errors/TOGGLE_DETAILS';
  */
 export function setError(response, json = null) {
   return async (dispatch) => {
-    const type = response.headers && response.headers.get('Content-Type');
-    const _json = !json && type === 'application/json'
-      ? await response.json() : null;
-    dispatch({
-      type: SET_ERROR,
-      status: response.statusCode,
-      statusText: response.statusText,
-      json: _json,
-    });
+    if (response.status) {
```

well, this is sort of problematic, because if it's not a response object with status this just swallows the error call?